nano/node/nodeconfig.cpp

```diff
@@ -577,7 +577,8 @@ bool nano::node_config::upgrade_json (unsigned version_a, nano::jsonconfig & jso
 		}
 		case 17:
 		{
-			json.put ("vote_generator_delay", vote_generator_delay.count ()); // Update value
+			json.put ("active_elections_size", 10000); // Update value
```

Comment:
Do we need it? If right now it's toml

Reply:
Otherwise will remain 50k during the v19->v20 upgrade, this forces the change one last time.